private/byte.rkt

```diff
@@ -8,7 +8,14 @@
   [byte-clear-leftmost-bits (-> byte? (integer-in 0 8) byte?)]
   [byte-clear-rightmost-bits (-> byte? (integer-in 0 8) byte?)]
   [byte-drop-rightmost-bits (-> byte? (integer-in 0 8) byte?)]
-  [byte-ref (-> byte? (integer-in 0 7) bit?)]))
+  [byte-ref (-> byte? (integer-in 0 7) bit?)]
+  [byte-and (-> byte? byte? byte?)]
+  [byte-or (-> byte? byte? byte?)]
+  [byte-not (-> byte? byte?)]
+  [byte-xor (-> byte? byte? byte?)]
+  [byte-nand (-> byte? byte? byte?)]
+  [byte-nor (-> byte? byte? byte?)]
+  [in-byte (-> byte? sequence?)]))
```

Comment:
For `in-byte`, could you strengthen the contract from `sequence?` to `(sequence/c bit?)`?

Reply:
I've strengthened the contract to (sequence/c bit?), and also added a #:min-count argument with a value of 8, being that bytes are always 8-bits (and thus values) long. Let me know if you don't think the min-count arg is suitable.
Thanks.

Reply:
That's perfect, thanks!